modules/claims_api/app/controllers/claims_api/v2/blueprints/power_of_attorney_request_blueprint.rb

```diff
@@ -4,97 +4,62 @@ module ClaimsApi
   module V2
     module Blueprints
       class PowerOfAttorneyRequestBlueprint < Blueprinter::Base
-        class Veteran < Blueprinter::Base
-          transform Transformers::LowerCamelTransformer
-
-          fields(
-            :first_name,
-            :middle_name,
-            :last_name
-          )
-        end
-
-        class Representative < Blueprinter::Base
-          transform Transformers::LowerCamelTransformer
-
-          fields(
-            :first_name,
-            :last_name,
-            :email
-          )
+        view :index do
+          field :id do |request|
+            request['id']
+          end
+
+          field :type do
+            'power-of-attorney-request'
+          end
+
+          field :attributes do |request|
+            {
+              veteran: {
+                first_name: request['vetFirstName'],
+                last_name: request['vetLastName'],
+                middle_name: request['vetMiddleName']
+              },
+              claimant: {
+                city: request['claimantCity'],
+                country: request['claimantCountry'],
+                military_po: request['claimantMilitaryPO'],
+                military_postal_code: request['claimantMilitaryPostalCode'],
+                state: request['claimantState'],
+                zip: request['claimantZip']
+              },
+              representative: {
+                poa_code: request['poaCode'],
+                vso_user_email: request['VSOUserEmail'],
+                vso_user_first_name: request['VSOUserFirstName'],
+                vso_user_last_name: request['VSOUserLastName']
+              },
+              received_date: request['dateRequestReceived'],
+              decision_date: request['dateRequestActioned'],
+              status: request['secondaryStatus'],
+              declined_reason: request['declinedReason'],
+              change_address_authorization: request['changeAddressAuth'],
+              health_info_authorization: request['healthInfoAuth']
+            }
```

Comment:
cf. BGS response: https://github.com/department-of-veterans-affairs/vets-api/blob/d870a139bfe0a634a982193dd7bb8d053b7b64f1/modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json#L416-L439